### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by MayamaTakeshi**
bla bla bla

### CODE REVIEW

File: f1.txt [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+c
+d
+e
+f
+g
+h
 I
+j
 J
+K
```
Comment (MayamaT) [inline f1.txt:11]: Shouldn't this be KK?

File: f1.txt [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+L
+M
```
Comment (MayamaT) [inline f1.txt:13]: Shouldn't this be MM?

**Review by MayamaT — COMMENTED** [review]
bla

**Comment by MayamaTakeshi** [comment]
No. Values are correct.

**Pull request merged**